tools/testing/test_selections.py

```diff
@@ -147,12 +145,28 @@ def shard(
         assert (
             unknown_tests == [] or serial
         ), f"Attmempting to parallelize unknown tests {unknown_tests}"
+        pytest_sharded_tests = sorted(
+            get_with_pytest_shard(known_tests, test_file_times, test_class_times),
+            key=lambda j: j.get_time(),
+            reverse=True,
+        ) + get_with_pytest_shard(unknown_tests, test_file_times, test_class_times)
+    else:
+        pytest_sharded_tests = get_with_pytest_shard(
+            tests, test_file_times, test_class_times
+        )
     del tests
 
-    known_tests = get_with_pytest_shard(known_tests, test_file_times, test_class_times)
+    round_robin_index = 0
 
-    if sort_by_time:
-        known_tests = sorted(known_tests, key=lambda j: j.get_time(), reverse=True)
+    def _get_min_sharded_job(
+        test: ShardedTest, sharded_jobs: List[ShardJob]
+    ) -> ShardJob:
+        if test.get_time() == 0:
```

Comment:
What do you think about default the test time to 60s, the same as the slow test threshold?  If that works out, this function could then be simplified using a min heap https://docs.python.org/3/library/heapq.html with `get_total_time` as the sorted value.

Reply:
There would have to be a distinction between the assumed 60s and the actual value of "None" for threshold setting in run_test, but that's easily solved by having a different function for getting the assumed time
What is the expected behavior if you want to sort by time and assume the time if its unknown?  Do the unknown times get put before or after the tests with <60s run time?

Reply:
Pushed a new commit doing this, putting the unknown tests behind the <60s tests, but didn't bother with the heap thing since the current format is alright imo.  We only have 6 shards in CI rn so it wouldn't be much faster

Reply:
undid this, pretty sure it was causing uneven sharding b/c there are test files that simply don't get run but do get collected